services/codecov/codecov.spec.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const { test, given } = require('sazerac')
+const Codecov = require('./codecov.service')
+
+const unknownCoverage = { coverage: 'unknown' }
+
+describe('Codecov', function() {
+  test(Codecov.prototype.transform, () => {
+    given({ json: {} }).expect(unknownCoverage)
+    given({ json: { commit: {} } }).expect(unknownCoverage)
+  })
```

Comment:
`sazerac` has a cool thing called `forCases` you can use here:

```js
test(Codecov.prototype.transform, () => {
  forCases([
    given({ json: {} }),
    given({ json: { commit: {} } }),
  ]).expect({ coverage: 'unknown' })
}
```

Reply:
👍